app/Main.hs

```diff
@@ -56,13 +56,13 @@ data Command
   -- | Run the project with some module, default Main
   | Run (Maybe ModuleName) BuildOptions
 
-  -- | Bundle the project, with optional main and target path arguments
+  -- | Bundle the project into an executable
   --   Builds the project before bundling
-  | Bundle (Maybe ModuleName) (Maybe TargetPath) NoBuild BuildOptions
+  | BundleApp (Maybe ModuleName) (Maybe TargetPath) NoBuild BuildOptions
```

Comment:
Let's not remove entirely the old `Bundle` and `MakeModule` constructors, but let's have them just call `die` with the suggestion to call the new version of the command instead